modules/statics/src/networks.ts

```diff
@@ -1013,6 +1001,30 @@ class ZkSyncTestnet extends Testnet implements EthereumNetwork {
   walletImplementationAddress = '0x92db2759d1dca129a0d9d46877f361be819184c4';
 }
 
+class Bera extends Mainnet implements EthereumNetwork {
+  name = 'Bera';
+  family = CoinFamily.BERA;
+  explorerUrl = '';
+  accountExplorerUrl = '';
+  chainId = 0;
+  nativeCoinOperationHashPrefix = '';
+  tokenOperationHashPrefix = '';
+}
+
+class BeraTestnet extends Testnet implements EthereumNetwork {
+  name = 'BeraTestnet';
+  family = CoinFamily.BERA;
+  explorerUrl = 'https://bartio.beratrail.io/tx/';
+  accountExplorerUrl = 'https://bartio.beratrail.io/address/';
+  chainId = 80084;
+  nativeCoinOperationHashPrefix = '80084';
+  tokenOperationHashPrefix = '80084-ERC20';
+  forwarderFactoryAddress = '0xccadf4198de4957d07d80673684ed19310579fc4';
+  forwarderImplementationAddress = '0xc75fb848eb0bb04e3faedbee0fc8144e8d62b83d';
```

Comment:
I think these are contracts deployed from the metamask address? You need to add contracts deployed using multisig-v4 repo

Reply:
deploying the new contracts

Reply:
As discussed we will update the contracts after deploying them